### PR TITLE
Improve Test Coverage for packages/web3-wallet/src

### LINKED ISSUE #458: Improve Test Coverage for packages/web3-wallet/src
This issue aims to improve the test coverage for this folder by:  
1. Identifying untested or under-tested functions and classes.  
2. Writing additional unit tests to cover edge cases and scenarios.

### PR DESCRIPTION
**Pull request by NueloSE**
resolves #458

### CODE REVIEW

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

File: packages/web3-wallet/src/password-crypto.test.ts [hunk 1]
```diff
@@ -16,12 +16,93 @@ You should have received a copy of the GNU Lesser General Public License
...
+    it('should handle an empty message for encryption and decryption', () => {
+      const emptyMessage = ''
+      const encryptedPayload = encrypt(password, emptyMessage)
+      const decryptedMessage = decrypt(password, encryptedPayload)
+
+      expect(decryptedMessage).toBe(emptyMessage)
+    })
+
+    it('should handle large messages for encryption and decryption', () => {
+      const largeMessage = 'A'.repeat(1_000_000) // 1 MB of data
```
Comment (pragmaxim) [inline packages/web3-wallet/src/password-crypto.test.ts:93]: I'd be happy with `500_000` since this single test takes 700ms and in would add up a lot in the big picture, I guess that supporting 0.5MB messages is enough

**Comment by NueloSE** [comment]
changed @pragmaxim

**Comment by NueloSE** [comment]
okay ser will it be merged? @pragmaxim
